.dotnet/src/Custom/Chat/InternalPredictionContent.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
+namespace OpenAI.Chat;
+
+// CUSTOM: Added base type.
+
+[CodeGenModel("PredictionContent")]
+internal partial class InternalPredictionContent : ChatOutputPrediction
```

Comment:
I don't see a generated file for this class. I believe this file can be deleted?